util/drakeUtil.h

```diff
@@ -45,6 +46,45 @@ template <typename Derived> inline void destroyDrakeMexPointer(const mxArray* mx
 //  mexPrintf(mexIsLocked() ? "mex file is locked\n" : "mex file is unlocked\n");
 }
 
+template <typename DerivedA>
+mxArray* eigenToMatlab(const DerivedA &m)
```

Comment:
in general, we don't want to have templated method implementations in header files.  in this case, since it's almost a macro (no cross dependence between different object files), it *might* be ok?